freqtrade/data/entryexitanalysis.py

```diff
@@ -278,9 +284,11 @@ def print_results(
             for ind in indicator_list:
                 if ind in res_df:
                     available_inds.append(ind)
-            ilist = ["pair", "enter_reason", "exit_reason"] + available_inds
+
+            merged_df = _merge_dfs(res_df, exit_df, available_inds)
```

Comment:
The only slight question / doubt i have now.... assuming a user uses `--indicator-list a b c` - he'd get 6 columns (`a (entry)`, `a (exit)`, ...).
but it's forced upon the user, right? so if he cares about `a` for entries - but `c` for exits - that currently can't be filtered.

It's not a problem (don't get me wrong) - it can just cause ... odd scenarios - if you look at 2 indicators for entry, and 3 different ones for exit - you'll get 10 colums (actually 14 - with default columns) - while you only needed 5 (actually 9 - with default columns)

Reply:
> But again do we really need this?

I'm absolutely not sure 😆 

for me, it'll work perfectly fine if we keep it as is at this currently in this PR - and _eventually_ refactor / enhance later - _should_ we get reports / feedback that it's causing problems.

Reply:
I would prefer option 1 as it's explicit from the argument as to the expected input

Reply:
Hmm having three separate but interacting options sounds a little complicated ... On second thought, the simplest option would be that `indicator-list` stays as before, and the default behaviour is to show both entry and exit for that indicator. Maybe then another option to filter by `--entry-only` and `--exit-only` might work?

Reply:
well that's essentially how it works now (without the "--entry-only"... options).